# 100.20231220-bunp-all-net7.0 CI nuget client log

100.20231220-bunp-all-net7.0.ci-nuget-client-log.md

Since `net7.0` TFM was addded nuget push/publish logs reveal errors for SOME packages.

Investigation is ongoing.

Packages seem to be published despite errors.

https://www.nuget.org/packages/Square.OkIO.JVM/3.6.0.1


Published `Square.OkIO.JVM.3.6.0.1`:

https://www.nuget.org/packages/Square.OkIO.JVM/

https://www.nuget.org/packages/Square.OkIO.JVM/3.6.0.1

```
2023-12-23T17:18:47.0177591Z Pushing Square.OkIO.JVM.3.6.0.1.nupkg to 'https://www.nuget.org/api/v2/package'...
2023-12-23T17:18:47.0179477Z Response status code does not indicate success: 413 (The package file exceeds the size limit. Please try again.).
2023-12-23T17:18:47.0180802Z System.Net.Http.HttpRequestException: Response status code does not indicate success: 413 (The package file exceeds the size limit. Please try again.).
2023-12-23T17:18:47.0181749Z    at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
2023-12-23T17:18:47.0182782Z    at NuGet.Protocol.Core.Types.PackageUpdateResource.EnsureSuccessStatusCode(HttpResponseMessage response, Nullable`1 codeNotToThrow, ILogger logger)
2023-12-23T17:18:47.0183870Z    at NuGet.Protocol.Core.Types.PackageUpdateResource.<>c__DisplayClass23_0.<PushPackageToServer>b__0(HttpResponseMessage response)
2023-12-23T17:18:47.0184808Z    at NuGet.Protocol.HttpSource.<ProcessResponseAsync>d__18`1.MoveNext()
2023-12-23T17:18:47.0185797Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0186611Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0187480Z    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2023-12-23T17:18:47.0188226Z    at NuGet.Protocol.Core.Types.PackageUpdateResource.<PushPackageToServer>d__23.MoveNext()
2023-12-23T17:18:47.0189020Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0189651Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0190351Z    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2023-12-23T17:18:47.0191072Z    at NuGet.Protocol.Core.Types.PackageUpdateResource.<PushPackageCore>d__20.MoveNext()
2023-12-23T17:18:47.0191862Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0298238Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0298975Z    at NuGet.Protocol.Core.Types.PackageUpdateResource.<PushPackage>d__19.MoveNext()
2023-12-23T17:18:47.0300410Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0301260Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0301986Z    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2023-12-23T17:18:47.0302721Z    at NuGet.Protocol.Core.Types.PackageUpdateResource.<Push>d__14.MoveNext()
2023-12-23T17:18:47.0303493Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0304096Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0304802Z    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2023-12-23T17:18:47.0305458Z    at NuGet.Commands.PushRunner.<Run>d__0.MoveNext()
2023-12-23T17:18:47.0306167Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0307002Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0307749Z    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2023-12-23T17:18:47.0308492Z    at NuGet.CommandLine.PushCommand.<ExecuteCommandAsync>d__36.MoveNext()
2023-12-23T17:18:47.0309529Z --- End of stack trace from previous location where exception was thrown ---
2023-12-23T17:18:47.0310193Z    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2023-12-23T17:18:47.0310939Z    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2023-12-23T17:18:47.0312408Z    at NuGet.CommandLine.Command.Execute()
2023-12-23T17:18:47.0318158Z    at NuGet.CommandLine.Program.MainCore(String workingDirectory, String[] args)
2023-12-23T17:18:47.0319036Z   PUT https://www.nuget.org/api/v2/package/
2023-12-23T17:18:47.0319957Z WARNING: Readme missing. Go to https://aka.ms/nuget-include-readme learn How to include a readme file within the package.
2023-12-23T17:18:47.0320823Z   Created https://www.nuget.org/api/v2/package/ 499ms
2023-12-23T17:18:47.0321371Z Your package was pushed.
```

Not published `Xamarin.TensorFlow.Lite.Select.TF.Ops`:

https://www.nuget.org/packages/Xamarin.TensorFlow.Lite.Select.TF.Ops

```
2023-12-23T17:18:47.1529896Z Pushing Xamarin.TensorFlow.Lite.Select.TF.Ops.2.14.0.1.nupkg to 'https://www.nuget.org/api/v2/package'...
2023-12-23T17:18:47.1531004Z   PUT https://www.nuget.org/api/v2/package/
2023-12-23T17:18:47.1532006Z   RequestEntityTooLarge https://www.nuget.org/api/v2/package/ 8739ms
2023-12-23T17:18:47.1668852Z ##[error]The nuget command failed with exit code(1) and error(Response status code does not indicate success: 413 (The package file exceeds the size limit. Please try again.).
System.Net.Http.HttpRequestException: Response status code does not indicate success: 413 (The package file exceeds the size limit. Please try again.).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at NuGet.Protocol.Core.Types.PackageUpdateResource.EnsureSuccessStatusCode(HttpResponseMessage response, Nullable`1 codeNotToThrow, ILogger logger)
   at NuGet.Protocol.Core.Types.PackageUpdateResource.<>c__DisplayClass23_0.<PushPackageToServer>b__0(HttpResponseMessage response)
   at NuGet.Protocol.HttpSource.<ProcessResponseAsync>d__18`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.Protocol.Core.Types.PackageUpdateResource.<PushPackageToServer>d__23.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.Protocol.Core.Types.PackageUpdateResource.<PushPackageCore>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at NuGet.Protocol.Core.Types.PackageUpdateResource.<PushPackage>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.Protocol.Core.Types.PackageUpdateResource.<Push>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.Commands.PushRunner.<Run>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.CommandLine.PushCommand.<ExecuteCommandAsync>d__36.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.CommandLine.Command.Execute()
   at NuGet.CommandLine.Program.MainCore(String workingDirectory, String[] args))
2023-12-23T17:18:47.1687983Z ##[section]Finishing: NuGet custom
```
